test/test_nn.py

```diff
@@ -6816,7 +6816,7 @@ def test_gaussian_nll_loss_reduction_modes(self):
                          F.gaussian_nll_loss(input, target, var, reduction='sum'))
         self.assertEqual(torch.mean(component_wise_loss),
                          F.gaussian_nll_loss(input, target, var, reduction='mean'))
-        with self.assertRaisesRegex(ValueError, 'is not valid'):
+        with self.assertRaisesRegex(ValueError, 'total is not a valid value for reduction'):
```

Comment:
I think this should be an ErrorInput?

Reply:
(actually, it seems so redundant with the `abc` ErrorInput below maybe it can just be deleted?)